packages/Frontend/src/components/NewJournal.jsx

```diff
@@ -1,5 +1,68 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Button } from 'semantic-ui-react';
+import ReactQuill from 'react-quill';
+import styled from 'styled-components';
+import 'react-quill/dist/quill.snow.css';
 
-export default function NewJournal() {
-  return (<h3>Create New Journal</h3>);
+const StyledDiv = styled.div`
+  height: 90%;
+  display: block;
+  position: relative;
+  .quill {
+    height: 90%;
+  }
+`;
+
+export default class NewJournal extends Component {
+  static modules = {
+    toolbar: [
+      [{ header: [1, 2, false] }],
+      ['bold', 'italic', 'underline', 'strike'],
+      [{ color: [] }, { background: [] }],
+      [{ align: [] }, { list: 'ordered' }, { list: 'bullet' }],
+      ['link', 'image', 'video'],
+    ],
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { text: '' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(value) {
+    this.setState({ text: value });
+  }
+
+  render() {
+    return (
+      <div style={{
```

Comment:
Let's try again to combine this with the styled div above, but if we can't this should be made a styled component by itself.